src/migrateCalculatedMeasure.ts

```diff
@@ -0,0 +1,29 @@
+interface LegacyCalculatedMeasure {
```

Comment:
I wouldn't put the `id` as part of this interface: it's generated on the fly, so there's no need to attach it.
Also, you should export it and reuse it as much as possible.